guide/how-it-works/lightning-service-providers.md

```diff
@@ -0,0 +1,310 @@
+   layout = "float-right"
+%}
+
+Lightning wallets can not receive payments without inbound liquidity in a channel. This is primarily the case with new wallets, but may also occur when channels are imbalanced on funded wallets.
+
+Offering inbound liquidity with a channel open service addresses this situation. When a user receives a payment without enough inbound liquidity, a channel is automatically opened. The network fee, and sometimes an additional service fee, for opening the channel is typically deducted from the payment amount.
+
+A channel open with inbound liquidity allows users to fund their wallets with a lightning payment, or combined with a [swap-in]({{ '/guide/how-it-works/lightning-service-providers/#swap-in' | relative_url }}) or [dual funding]({{ '/guide/how-it-works/lightning-service-providers/#dual-funding' | relative_url }}) service can allow funding via on-chain.
+
+[Blocktank](https://blocktank.synonym.to/) and [Dunder](https://github.com/hsjoberg/dunder-lsp) are two open source LSP clients that help applications offer inbound liquidity to their users.
+
+[Liquidity ads](https://medium.com/blockstream/setting-up-liquidity-ads-in-c-lightning-54e4c59c091d) are a protocol-level way for anyone to act as an LSP and advertise their willingness to offer liquidity to other users.
+
+</div>
+
+### Dual-funding
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/dual-funding.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/dual-funding@2x.jpg"
+   alt-text = "Blocks representing unconfirmed spend."
+   width = 140
+   height = 140
+   layout = "float-right"
+%}
+
+Using a channel open service requires a LSP to open a channel for the user on their behalf, which requires trust. An LSP could refuse to open a channel for a user or require invasive privacy practices, leading to application-level censorship.
+
+A [dual-funding](https://bitcoinops.org/en/topics/dual-funding/) channel open service solves this issue of trust by having users cooperatively open a channel with the LSP. Users get the benefits of having both inbound and outbound liquidity without having to trust the LSP.
+
+Dual funding is only applicable when funding a wallet with on-chain bitcoin. The issue of trusting the LSP still exists if funding with a lightning payment.
+
+</div>
+
+### Spend-unconfirmed
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/spend-unconfirmed.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/spend-unconfirmed@2x.jpg"
+   alt-text = "Blocks representing unconfirmed spend."
+   width = 140
+   height = 140
+   layout = "float-right"
+%}
+
+The bitcoin in a payment channel can not be spent until the on-chain transaction that opened the channel is confirmed. If the Mempool is congested, this could take a long time.
```

Comment:
Good place to link to glossary. Also I don't think there's the mempool, only a mempool, right?
```suggestion
The bitcoin in a payment channel can not be spent until the on-chain transaction that opened the channel is confirmed. If a [mempool]({{ '/guide/nboarding/backing-up-a-wallet/cloud-backup/' | relative_url }}) is congested, this could take a long time.
```

Reply:
I always hear people refer to it as "the mempool", even though technically each node has its own mempool. Regardless, lowercase is probably in order.

Reply:
The glossary refers to it as "the mempool". Let's be consistent.

Reply:
Also, the URL in the suggestion doesn't go to the glossary.

Reply:
```suggestion
The bitcoin in a payment channel can not be spent until the on-chain transaction that opened the channel is confirmed. If the [mempool]({{ '/guide/glossary/#mempool' | relative_url }}) is congested, this could take a long time.
```

Reply:
To avoid complexity, we could just say if the network is congested